modules/core/src/main/java/org/apache/ignite/mxbean/CacheMetricsMXBean.java

```diff
@@ -151,10 +151,18 @@
     @MXBeanDescription("Number of non-null values in the cache.")
     public int getSize();
 
+    /** {@inheritDoc} */
+    @MXBeanDescription("Number of non-null values in the cache.")
```

Comment:
Is issue tell you to add your metrics to spring? If yes maybe to add to the description "as a long type"?

Reply:
Done.